ydb/core/tablet_flat/flat_part_iter_multi.h

```diff
@@ -1247,7 +1250,8 @@ namespace NTable {
         mutable TSmallVec<TPartGroupRowIt> Groups;
 
         // Key is lazily initialized so needs to be mutable
-        mutable TSmallVec<TCell> Key;
```

Comment:
Using TSmallVec is an important optimization, since we often have a very small number of key columns (e.g. 1) and we'd like to avoid extra allocations in those cases.

Reply:
Updated.